CHANGELOG.md

```diff
@@ -1,19 +1,22 @@
 # Changelog
 
-## Unreleased
+## [0.14.0]
 
 ### Added
 
-- Documentation for development environment setup on AWS.
+- Documentation for development environment setup on AWS in
+  `dev-machine-setup.md`.
+- Documentation for microVM networking setup in `docs/network-setup.md`.
 - Limit the maximum supported vCPUs to 32.
 
 ### Changed
 - Log the app version when the `Logger` is initialized.
 - Pretty print panic information.
 - Firecracker terminates with exit code 148 when a non-whitelisted syscall
-is intercepted.
+  is intercepted.
 
 ### Fixed
+- Building with the `vsock` feature is verified in the CI suite.
```

Comment:
I wouldn't add the CI part as we don't really mention CI changes in the changelog.
"Fixed build with `vsock` feature." should suffice.

Reply:
Done